test/test_jit.py

```diff
@@ -11513,7 +11504,9 @@ def f(x, y):
         y = torch.randn(4, 4, dtype=torch.float, device='cuda')
 
         ge = self.checkTrace(f, (x, y))
-        self.assertExpectedGraph(ge.graph_for(x, y))
+        graph = ge.graph_for(x, y)
+        FileCheck().check("broadcast_tensors").check('FusionGroup_0') \
```

Comment:
For test_chunk_distributes_cuda:

What we want to check is the following:
(1) broadcast_tensors happens before FusionGroup_0 gets called
(2) There are 2 prim::ConstantChunk inside the FusionGroup_0 definition

If I'm understanding this correctly, this FileCheck expression checks (1) but doesn't explicitly check (2): it would also match the following
```
blah = aten::broadcast_tensors(%2)	
blah = FusionGroup_0(%5, %4)
blah = ConstantChunk(...)
blah = ConstantChunk(...)

with prim::FusionGroup_0 = graph(
...
)
```

I think you can peek into the FusionGroup graph by using node.graph or node.subgraph?